keyboards/tweetydabird/lotus58/info.json

```diff
@@ -0,0 +1,150 @@
+{
+    "manufacturer": "Tweetys Wild Thinking",
+    "keyboard_name": "Lotus 58 Glow (QMK)",
+    "maintainer": "TweetyDaBird",
+    "bootloader_instructions": "Short marked pads on PCB, or hold top-outer key when plugging in each hand.",
+    "build": {
+        "lto": true
+    },
+   "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {
+                "pin_a": "F5",
+                "pin_b": "F4",
+                "resolution": 2
+            }
+        ]
+    },
+    "features": {
+        "audio": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "grave_esc": false,
+        "magic": false,
+        "mousekey": false,
+        "nkro": true,
+        "oled": true,
+        "rgblight": true,
+        "space_cadet": false,
+        "split": true,
+        "tap_dance": false,
+		"tri_layer": true
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B2", "B3", "B6", "F7", "F6"],
+        "rows": ["D4", "C6", "D7", "E6", "B4"]
+    },
+    "pin_compatible": "promicro",
+    "processor": "atmega32u4",
+    "rgblight": {
+        "brightness_steps": 17,
+        "hue_steps": 17,
```

Comment:
```suggestion
```

Reply:
Because some of these are the defaults, because other add little value, because you were asked to so that this PR can be submitted to the repo.

Reply:
Either of the first two reasons had been quite sufficient if they had been given to start with. The third statement is not a reason, but an opinion.

Reply:
I didn't argue an opinion. I asked the reason why, since I genuinely didn't know why you wanted them removed, since they wouldn't in any way be wrong. But if they simply mimimc what has become default since I added them a very long time ago, they should indeed be removed, and I did so once you gave the reason.

Yes, I have admittedly 'argued' or more discussed on other points where my opinion differed. If that is not allowed, feel free to point to that rule. Otherwise I will continue to ask questions if I don't understand the reasoning. Or in this case the reason wasn't given. It might be obvious to you, it just wasn't to me. 

No slight intended. Just a question.